From a7b0a65898ed10d109eaa10b34a9516c0c05f774 Mon Sep 17 00:00:00 2001
From: Ondrej Holy <oholy@redhat.com>
Date: Thu, 15 Dec 2016 11:18:00 +0100
Subject: [PATCH] network: Do not spam journal by useless warnings

The following warning is always printed from network backend
if smb backend is installed and network backend is mounted:
Couldn't create directory monitor on smb://[WORKGROUP]/.

You can see it with two different error messages. The first is
printed if we are not in samba environment, or the workgroup is
misconfigured:
Error: The specified location is not mounted

The second is printed in other cases, because monitoring is not
supported by smb backend:
Error: Operation not supported by backend

This isn't really useful and just spams the journal, let's use
g_debug instead of g_warning.
---
 daemon/gvfsbackendnetwork.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/gvfsbackendnetwork.c b/daemon/gvfsbackendnetwork.c
index fc6cca47..95fee1d6 100644
--- a/daemon/gvfsbackendnetwork.c
+++ b/daemon/gvfsbackendnetwork.c
@@ -358,7 +358,7 @@ recompute_files (GVfsBackendNetwork *backend)
           else
             {
 	      char *uri = g_file_get_uri (server_file);
-              g_warning ("Couldn't create directory monitor on %s. Error: %s", 
+              g_debug ("Couldn't create directory monitor on %s. Error: %s",
 	     		 uri, error->message);
 	      g_free (uri);
               g_clear_error (&error);
-- 
2.15.0

